CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Unreleased
 ----------
 
+17.1.0 (March 5, 2021)
+----------
+* Add warning to OmniAuth generator if shopify_provider exists to signal updates to OmniAuth configuration [#1188](https://github.com/Shopify/shopify_app/pull/1188)
+* Create OmniAuthConfiguration object to build future OmniAuth strategies [#1190](https://github.com/Shopify/shopify_app/pull/1190)
+* Add access scopes to Shop and User model generators [#1191](https://github.com/Shopify/shopify_app/pull/1191)
+* Store access scopes after successful OAuth with ShopSessionStorageWithScopes and UserSessionStorageWithScopes [#1192](https://github.com/Shopify/shopify_app/pull/1192)
+* Use access scopes strategies to handle changes to scopes requested by app [#1197](https://github.com/Shopify/shopify_app/pull/1197)
```

Comment:
I wonder if we should coalesce these into something friendlier for external developers - this may not make a lot of sense without the context behind the changes.

I'd consider saying something in the lines of (with more than a few grains of salt):
- Created OmniAuthConfiguration object to build future OmniAuth strategies
- Added access scopes to Shop and User models, added checks to handle scope changes

Reply:
Sounds good! Let me go ahead and update that